.changelog

```diff
@@ -4,6 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- SUSE Manager: Universe is not required for installing spacecmd
+  on Ubuntu (bsc1187337)
+
+# Last packaged: ?
```

Comment:
I am not sure what do do here. I think Joseph should have cleaned this up when he packaged last time. Anyway, please fix it on a separate PR :-)

Reply:
:+1: if nobody knows better, we can write `Last packaged: before 2021-08-18`.